qa/L0_results/test.sh

```diff
@@ -23,7 +23,7 @@ MODEL_ANALYZER="`which model-analyzer`"
 REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
 MODEL_REPOSITORY=${MODEL_REPOSITORY:="/mnt/nvdl/datasets/inferenceserver/$REPO_VERSION/libtorch_model_store"}
 CHECKPOINT_REPOSITORY=${CHECKPOINT_REPOSITORY:="/mnt/nvdl/datasets/inferenceserver/model_analyzer_checkpoints/2022_08_02"}
-QA_MODELS="vgg19_libtorch resnet50_libtorch"
+QA_MODELS="resnet50_libtorch"
```

Comment:
what happens before this change? Does the test fail? Why?
We should double check with Jakub that this change doesn't break Model Navigator. I'm pretty sure they never run the report step, but we should double check.